dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -584,7 +588,9 @@ boolean isUnusedClassSymbolReference(ClassSymbolReference reference) {
           constantPoolIndexForClass(sourceJavaClass, targetClassName);
       Verify.verify(
           !targetConstantPoolIndices.isEmpty(),
-          "The target class symbol reference is not found in source class");
+          "The target class %s is not found in source class %s",
```

Comment:
This message doesn't sound right unless target class is an inner class.

Reply:
Updated.